src/modules/menu/components/advancedTools/ImportFile/ImportFile.vue

```diff
@@ -0,0 +1,71 @@
+<script setup>
+import { ref } from 'vue'
+
+import ImportFileOnlineTab from '@/modules/menu/components/advancedTools/ImportFile/ImportFileOnlineTab.vue'
+import ImportFileLocalTab from '@/modules/menu/components/advancedTools/ImportFile/ImportFileLocalTab.vue'
+
+const selectedTab = ref('online')
+</script>
+
+<template>
+    <div data-cy="import-file-content">
+        <ul class="nav nav-tabs" role="tablist">
+            <li class="nav-item" role="presentation">
+                <button
+                    class="nav-link py-1"
+                    :class="{
+                        active: selectedTab === 'online',
+                    }"
+                    data-bs-toggle="tab"
+                    data-bs-target="#nav-online"
+                    type="button"
+                    role="tab"
+                    aria-controls="nav-online"
+                    :aria-selected="selectedTab === 'online'"
+                    data-cy="online-import-btn"
+                    @click="() => (selectedTab = 'online')"
+                >
+                    Online
+                </button>
+            </li>
+            <li class="nav-item" role="presentation">
+                <button
+                    class="nav-link py-1"
+                    :class="{
+                        active: selectedTab === 'local',
+                    }"
+                    data-bs-toggle="tab"
+                    data-bs-target="#nav-local"
+                    type="button"
+                    role="tab"
+                    aria-controls="nav-local"
+                    :aria-selected="selectedTab === 'local'"
+                    data-cy="local-import-btn"
+                    @click="() => (selectedTab = 'local')"
+                >
+                    Local
+                </button>
+            </li>
+        </ul>
+        <div class="tab-content mt-2">
+            <!-- Online Tab -->
+            <ImportFileOnlineTab
+                :class="{ active: selectedTab === 'online', show: selectedTab === 'online' }"
+            />
+            <!-- Local tab -->
+            <ImportFileLocalTab
+                :class="{ active: selectedTab === 'local', show: selectedTab === 'local' }"
+            />
+        </div>
```

Comment:
Wouldn't it be better to only load the component matching the current selection? Something like

```
<script setup>
const tabComponent = computed(() => {
    if (selectedTab === 'online') {
        return ImportFileOnlineTab
    }
    return ImportFileLocalTab
})
</script>
<template>
    <component :is="tabComponent" class="active show" />
<template>
```

Reply:
Would this not be less performant when switching back and forth on the tabs ? Also now if you write a url in online tab and switch to file and back to online what you typed is kept which is kind of a nice feature I think

Reply:
The memory of the URL is a nice feature yes, we can keep it this way